components/progressBar.js

```diff
@@ -0,0 +1,36 @@
+import { ProgressBar } from 'react-bootstrap';
+import { useState } from 'react';
+import { getCurrentIteration, checkFinish } from '../components/litmus-setup';
+import { getIterationNum } from './test-page-setup';
+let finish;
+let percentage = 0;
+let finalIter = 0;
+let finalPercentage = 0;
+let defaultState = true;
+export function resetProgressBar(){
+    finish = false; 
+    percentage = 0;
+}
+export function setProgressBarState(){
+    defaultState = false;
+}
+export default function progressBar(){
+    percentage = Math.floor(getCurrentIteration()*100/getIterationNum());
+    finalIter = getCurrentIteration()+1;
+    finalPercentage =  Math.floor(finalIter*100/getIterationNum());
```

Comment:
I'm a little confused about what this is used for. It seems like it's only used when the test run is done, in which case do we even need it?

Reply:
using the provided iteration num and current iteration to check whether the test is done or not